Fix comment indentation issue

tugsbayasgalan opened the pull request:
Stack from [ghstack](https://github.com/ezyang/ghstack):
* __->__ #70227

fixes: https://github.com/pytorch/pytorch/issues/65452
Differential Revision: [D33251107](https://our.internmc.facebook.com/intern/diff/D33251107)

tugsbayasgalan committed:
Fix comment indentation issue

[ghstack-poisoned]

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33251107).

gmagogsfm commented on test/test_jit.py:
```diff
@@ -9731,6 +9731,18 @@ def foo():
+    def test_comment_ignore_indent(self):
+        class Model(torch.nn.Module):
+            def __init__(self):
+    # ERROR  # noqa: E115
```
```suggestion
    # useless comment that is not indented correctly  # noqa: E115
```

tugsbayasgalan committed:
Update on "Fix comment indentation issue"


fixes: https://github.com/pytorch/pytorch/issues/65452
Differential Revision: [D33251107](https://our.internmc.facebook.com/intern/diff/D33251107)

[ghstack-poisoned]

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33251107).

tugsbayasgalan committed:
Update on "Fix comment indentation issue"


fixes: https://github.com/pytorch/pytorch/issues/65452
Differential Revision: [D33251107](https://our.internmc.facebook.com/intern/diff/D33251107)

[ghstack-poisoned]

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33251107).

tugsbayasgalan committed:
Update on "Fix comment indentation issue"


fixes: https://github.com/pytorch/pytorch/issues/65452
Differential Revision: [D33251107](https://our.internmc.facebook.com/intern/diff/D33251107)

[ghstack-poisoned]

tugsbayasgalan commented:
@tugsbayasgalan has imported this pull request.  If you are a Facebook employee, you can view this diff [on Phabricator](https://www.internalfb.com/diff/D33251107).
